display_list/testing/dl_rendering_unittests.cc

```diff
@@ -4258,8 +4256,12 @@ class DisplayListNopTest : public DisplayListCanvas {
       result_canvas->clear(SK_ColorTRANSPARENT);
       result_canvas->drawImage(test_image.get(), 0, 0);
       result_canvas->drawRect(test_bounds, sk_paint);
-      result_canvas->flush();
-      result_surface->sk_surface()->flushAndSubmit(true);
+      if (GrDirectContext* direct_context = GrAsDirectContext(
+              result_surface->sk_surface()->recordingContext())) {
+        direct_context->flushAndSubmit();
+        return direct_context->flushAndSubmit(result_surface->sk_surface(),
```

Comment:
Now also wondering if the `return` statement here and on 4324 are correct? Maybe @flar knows?

Reply:
It looks like they were removed already, but no they don't look right.